### ISSUE TITLE
Feature Request: custom notification status bar icons

### ISSUE THREAD

**Issue by wunter8** [binwiederhier/ntfy#367]

**Comment by binwiederhier** [comment]
I agree that this would be nice to have. Though it's not super high prio for me.

**Comment by ymau** [comment]
It would also be nice if the ntfy statusbar icon could be hidden while there are no active notifications.

**Comment by Iceman248** [comment]
This would be nice. I use the app "join" for some notifications I would like to migrate over to ntfy, but "join" uses the app icon that the notification was sent from instead of the "join" icon, which is nice to see with adaptive display when all I can see are the app icons or in the status bar.

**Comment by Nairou** [comment]
Came for the same thing. I'd like to migrate more of my phone notifications to ntfy, but it is hard to tell at a glance where the notification came from since they all share the same ntfy notification icon.

A good reference source for this could be the Home Assistant companion app, which allows the message data to specify any [MDI icon](https://pictogrammers.com/library/mdi/) and color to be used.

**Comment by wunter8** [comment]
Nothing yet unfortunately

**Comment by shale-aspect** [comment]
Really hoping for this too. I'm still using Home Assistant for some notifications because of the customization options, not only changing the icon, but also the color and [lockscreen visibility](https://companion.home-assistant.io/docs/notifications/notifications-basic/?_highlight=visibility#notification-sensitivity--lock-screen-visibility).

Here is the same lockscreen notification with Home assistant (private notification), Home assistant (public notification), and ntfy :

![notifications](https://github.com/binwiederhier/ntfy/assets/96342700/a06920a4-f06f-485e-b5d4-8f5df64e7997)

**Comment by sjs28** [comment]
Looks like this request has been lurking for a while. I've been using ntfy for nearly a year now and I love it, but the fact all my topics have the same icon drives me insane. I'd absolutely love to have this feature!
